docs/en/index.rst

```diff
@@ -1,4 +1,4 @@
-Welcome to Doctrine 2 ORM's documentation!
+Welcome to Doctrine 3 ORM's documentation!
```

Comment:
I think we should remove that number entirely

Reply:
Agreed.

Reply:
done